gradle.properties

```diff
@@ -1 +1 @@
-version=2.2.1-SNAPSHOT
+version=2.3.0-SNAPSHOT
```

Comment:
i'll deal with the merges to different branches.


On Mon, Feb 23, 2015 at 11:36 AM, David <notifications@github.com> wrote:

> In gradle.properties
> <https://github.com/nebula-plugins/gradle-dependency-lock-plugin/pull/46#discussion_r25193981>
> :
>
> > @@ -1 +1 @@
> > -version=2.2.1-SNAPSHOT
> > +version=2.3.0-SNAPSHOT
>
> Sure. Should I try to rebase this on the 1.8 branch?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nebula-plugins/gradle-dependency-lock-plugin/pull/46/files#r25193981>
> .
>